tensorflow/compiler/mlir/tensorflow/ir/tf_ops.td

```diff
@@ -2728,6 +2728,66 @@ def XlaSparseDenseMatmulGradWithFtrlAndStaticBufferSizeOp : TF_Op<"XlaSparseDens
   );
 }
 
+def TF_UniformDequantizeOp : TF_Op<"UniformDequantize", [Pure]> {
```

Comment:
I don't know much about the TF op infra, but why did this need to be moved from generated ops to here?

Reply:
We edited this definition to add the new quint8 type and it was moved here to honor this [comment](https://github.com/tensorflow/tensorflow/blob/b13548b8be8093bc6c742354cd2c56a16ac8f24d/tensorflow/compiler/mlir/tensorflow/ir/tf_generated_ops.td#L16-L24)